apps/public-docsite-resources/just.config.ts

```diff
@@ -6,5 +6,5 @@ preset();
 task('generate-json', () => generatePageJsonFiles(require('./config/api-docs')));
 
 // copied from scripts/just.config.js with addition of generate-json
-task('build', series('clean', 'copy', 'sass', 'generate-json', 'ts')).cached();
+task('build', series('clean', 'copy', 'sass', 'generate-json', 'ts')).cached!();
```

Comment:
whilst using `!` is deprecated and should be avoided, this is caused by bad just types, so sticking with `!` instead of nodejs standard `?`